Dockerfile

```diff
@@ -1,28 +1,21 @@
-FROM redis:3.0
+FROM redis:3.0-alpine
 MAINTAINER eyeos
-
 ENV WHATAMI redis
 
-COPY start.sh /tmp/start.sh
-
-RUN \
-	apt-get update && \
-	apt-get install -y curl dnsmasq && \
-	curl -sL https://deb.nodesource.com/setup_0.10 | bash - && \
-	apt-get install -y unzip nodejs build-essential git build-essential && \
-	chmod +x /tmp/start.sh && \
-	curl -L https://releases.hashicorp.com/serf/0.6.4/serf_0.6.4_linux_amd64.zip -o serf.zip && \
-	unzip serf.zip && \
-	mv serf /usr/bin/serf && \
-	npm install -g npm@2.14.4 && \
-	npm install -g eyeos-run-server eyeos-tags-to-dns eyeos-service-ready-notify-cli && \
-	apt-get clean && \
-	apt-get -y remove --purge curl git build-essential && \
-	apt-get -y autoremove && \
-	rm -rf /var/lib/apt/lists/*
+RUN apk update && \
+    apk add curl make gcc g++ git python unzip dnsmasq nodejs && \
+    npm install -g eyeos-run-server eyeos-tags-to-dns eyeos-service-ready-notify-cli && \
+    curl -L https://releases.hashicorp.com/serf/0.6.4/serf_0.6.4_linux_amd64.zip -o serf.zip && \
+    unzip serf.zip && mv serf /usr/bin/serf && rm -f serf.zip && \
+    apk del curl make gcc g++ git python unzip && \
+    npm remove -g npm && rm -r /root/.npm && \
+    rm -r /etc/ssl /var/cache/apk/*
 
 COPY dnsmasq.conf /etc/dnsmasq.d/
 COPY dnsmasq_generic.conf /etc/dnsmasq.conf
-CMD /tmp/start.sh
 
 EXPOSE 6379
+
+COPY start.sh /tmp/start.sh
```

Comment:
Could you please copy this to /var/service instead of /tmp/? It's more consistent.

This way you can even delete the contents of /tmp/*